tfx/components/evaluator/executor_test.py

```diff
@@ -75,6 +76,10 @@ def testEvalution(self, exec_properties):
         constants.BLESSING_KEY: [blessing_output],
     }
 
+    # List needs to be serialized before being passed into Do function.
+    exec_properties[constants.EXAMPLES_PATH_SPLITS_KEY] = json_utils.dumps(
+        ['eval'])
```

Comment:
could you verify if multiple splits works (no need to update this PR though, manually try ['train', 'eval'] to see if it works) ?

Reply:
yes, it works with multiple splits now, i tried locally